notebooks/official/tensorboard/tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -0,0 +1,999 @@
+{
```

Comment:
<p>No longer need to assign an endpoint. Endpoint is automatically set in the SDK initialization. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>